brave/brave-core
# Backup actions are now functional

Fixes brave/brave-browser#929

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Start Brave and create wallet
2. Click advance settings (backup/restore wallet button)
3. On backup wallet page click copy button
4. Open text editor and verify that backup keys are properly pasted.

*

1. Start Brave and create wallet
2. Click advance settings (backup/restore wallet button)
3. On backup wallet page click save button
4. Save and open file and verify the backup keys are properly saved.
5. Try to recover wallet with this file

*

1. Start Brave and create wallet
2. Click advance settings (backup/restore wallet button)
3. On backup wallet page click print button
4. Verify print preview screen shows backup keys correctly and print if necessary


## Reviewer Checklist:
